tests/infrastructure/instance_types/conftest.py

```
@@ -174,3 +171,20 @@ def windows_vm_for_dedicated_cpu(request, unprivileged_client, namespace, latest
     ) as vm:
         vm.start()
         yield vm
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)